### PR TITLE
feat(api) : deployment & property service

### PR DESCRIPTION
**Pull request by soumyadip007**
## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/api/.prettierrc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 {
   "singleQuote": true,
-  "printWidth": 100,
+  "printWidth": 150,
```
Comment (hybridx) [inline packages/api/.prettierrc:4]: this could be too long

File: packages/api/.vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (hybridx) [inline packages/api/.vscode/extensions.json:1]: @soumyadip007 this file could be removed

**Review by hybridx — APPROVED** [review]
minor changes rest LGTM

File: packages/api/.prettierrc (hunk 1 above)
Comment (soumyadip007) [reply packages/api/.prettierrc:4]: Actually if we keep it hundred the number of lines will be increased a lot, so I kept it 150.

File: packages/api/.vscode/extensions.json (hunk 2 above)
Comment (soumyadip007) [reply packages/api/.vscode/extensions.json:1]: Thanks, will add that in gitignore too

**Comment by soumyadip007** [comment]
@hybridx I've removed the extension.json

**Pull request merged**